content/shmem_lock_pe.tex

```diff
@@ -0,0 +1,69 @@
+\apisummary{
+    Releases, locks, and tests a mutual exclusion memory lock on a specific \ac{PE}.
+}
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_clear_lock_pe(volatile long *lock, int pe);
+void shmem_set_lock_pe(volatile long *lock, int pe);
+int shmem_test_lock_pe(volatile long *lock, int pe);
+\end{Csynopsis}
+
+\begin{Fsynopsis}
+INTEGER pe, lock, SHMEM_TEST_LOCK
+CALL SHMEM_CLEAR_LOCK_PE(lock, pe)
+CALL SHMEM_SET_LOCK_PE(lock, pe)
+I = SHMEM_TEST_LOCK_PE(lock, pe)
+\end{Fsynopsis}
+
+\begin{apiarguments}
+\apiargument{IN}{lock}{A symmetric data object that is a scalar variable or an array
+    of  length \CONST{1}.  This data  object  must  be set to \CONST{0} on all
+    \ac{PE}s prior to the first use.  \VAR{lock}  must  be  of type \CONST{long}.
+    If you are using \Fortran, it must be of default kind.}
+\apiargument{IN}{pe}{An integer that indicates the \ac{PE} number upon which lock is
```

Comment:
Request: Revisit wording here to be less Fortran-y.

Suggest: "lock: A pointer to a symmetric data object of type \CTYPE{long}. This data object must be initialized to \CONST{0} on the target PE prior to first use."

(See #55 for `\CTYPE`.)